analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -29,7 +29,6 @@ packages:
   - "curl"
   declared_licenses_processed:
     spdx_expression: "curl"
```

Comment:
commit-message: 
- "is map" -> "is mapped"
- Please wrap lines after 75? chars